migration: add callout for rescheduling mechanism (#6337)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/migration.md

## Recommended actions

:::info Rescheduling your migration

> Customers on dbt Cloud Enterprise tiers can postpone their account migration by up to 45 days. To reschedule your migration, go to **Account Settings** → **Migration guide**.

Reviewer: Just to keep the language consistent. We generally use 2nd person.
```suggestion
If you're on the dbt Cloud Enterprise tier, you can postpone your account migration by up to 45 days. To reschedule your migration, navigate to **Account Settings** → **Migration guide**.
```

> For help, contact us at [sales@getdbt.com](mailto:sales@getdbt.com).

Reviewer 2: Should this be our support email instead, or is sales appropriate for Enterprise accounts?

Author: @aaronraff I believe this should be `support@getdbt.com`.

> For help, contact us at [support@getdbt.com](mailto:support@getdbt.com).

Reviewer: ```suggestion
For help, contact the dbt Support Team at [support@getdbt.com](mailto:support@getdbt.com).
```
